ProjectSettings/ProjectSettings.asset

```diff
@@ -733,7 +732,7 @@ PlayerSettings:
     UNet: 0
     Unity_Ads: 0
   facebookSdkVersion: 7.9.1
-  apiCompatibilityLevel: 2
+  apiCompatibilityLevel: 3
```

Comment:
What is compatibility level 3? I only have one possible value.
![image](https://user-images.githubusercontent.com/3580640/37315162-6ff770e4-2615-11e8-8b9b-934c04e79bc8.png)

Reply:
because that's the enum value

Reply:
Right, but enum value 2 (and 0, and 1, and probably any number since I only have one option) was .NET 4.6 for me too. I'm curious which version of Unity seemingly has more than one value here.

Reply:
Unsure, but maybe the editor is doing something to ensure the compatibility level is set to this value because it's using the 4.6 scripting runtime version.

Reply:
Yeah, must be